File: ex2.2/ex.c
#include <stdio.h>

int main (void) {
	int n1, n2;
	printf("Digite n1: ");
	scanf("%d",&n1);
	printf("Digite n2: ");
	scanf("%d",&n2);
	if (n1<n2) {
		while (n1<n2-1) {
			n1++;
			printf("%d\n",n1);
		}
	}
	else {
		if (n2<n1) {
			while (n2<n1-1) {
				n2++;
				printf("%d\n",n2);
			}
		}
	}
	return 0;
}

File: ex2.2/ex.py
def le():
	while True:
		try:
			n1=int(input('Digite n1: '))
			break
		except:
			print 'Dado invalido, tente novamente.'
	while True:
		try:
			n2=int(input('Digite n2: '))
			break
		except:
			print 'Dado invalido, tente novamente.'
	return n1,n2
def impr(n1,n2):
	if n1<n2:
		while n1<n2-1:
			n1=n1+1
			print n1
	elif n2<n1:
		while n2<n1-1:
			n2=n2+1
			print n2
	return
n1,n2=le()
impr(n1,n2)
